docs/docs/1.7.x/security/api-server-auth.md

```diff
@@ -1,156 +1,167 @@
-# API Server Authentication
+# API server authentication
 
-By default, Kuma exposes API Server on [ports](../networking/networking.md) `5681` and `5682` (protected by builtin TLS). This server is used for actions like
-* Accessing policies and objects
-* Managing policies and objects (on Universal)
-* Accessing GUI
+Kuma exposes API server on [ports](../networking/networking.md) `5681` and `5682` (protected by TLS).
 
-Authenticated user can be authorized to execute administrative actions such as
-* Managing Kuma Secrets (on Universal)
-* Generating Dataplane Tokens
-* Generating User Tokens
+An authenticated user can be authorized to execute administrative actions such as
+* Managing administrative resources like Kuma Secrets on Universal
+* Generating user token, data plane proxy token, zone ingress token, zone token
 
-## User Token
+## User token
 
-User Token is a JWT token that contains
+A user token is a signed JWT token that contains
 * The name of the user
-* The list of groups that user belongs to
+* The list of groups that a user belongs to
 * Expiration date of the token
 
-The User Token is signed by a signing key that is autogenerated on the control plane. The signing key is SHA256 encrypted.
+### Groups
 
-You can check for the signing key:
+A user can be a part of many groups. Kuma adds two groups to a user automatically:
+* Every authenticated user is a part of `mesh-system:authenticated`.
+* Every user that doesn't provide authentication data is a part of `mesh-system:unauthenticated`.
 
-```sh
-kumactl get global-secrets
-```
-which returns something like:
-```
-NAME                          AGE
-user-token-signing-key-1   36m
-```
-
-## Implicit groups
+### Admin user token
 
-A user can be a part of many groups. On top of that, Kuma adds two groups automatically.
-* Every authenticated user is a part of `mesh-system:authenticated`
-* Every user that do not provide authentication data is a part of `mesh-system:unauthenticated`.
-
-### Usage
+Kuma creates an admin user token when the control plane starts.
+The admin user token is a user token issued for user `mesh-system:admin` that belongs to `mesh-system:admin` group.
+This group is [by default authorized](./api-access-control.md) to execute all administrative operations.
```

Comment:
```suggestion
This group is [authorized by default](./api-access-control.md) to execute all administrative operations.
```

It sounds a bit more natural this way (at least to me) - maybe some native speakers can chip in (@michaelbeaumont @parkanzky).